### PR TITLE
fix(TBD-14994): create hbase table with split Keys regions

### PR DESCRIPTION
**Pull request by sanulich**

### CODE REVIEW

File: main/components_lib/talend-bigdata-core-di/src/main/java/org/talend/bigdata/core/di/components/hbase/THbaseTable.java [hunk 1]
```diff
@@ -105,6 +108,20 @@ default List<HColumnDescriptor> getColumnFamilies() {
         return descriptorList;
     }
 
+    default byte[][] convertRegionSplitKeys(){
```
Comment (lbourgeois) [inline main/components_lib/talend-bigdata-core-di/src/main/java/org/talend/bigdata/core/di/components/hbase/THbaseTable.java:111]: I can't see where this method is used except in unit tests

File: main/plugins/org.talend.designer.components.bigdata/components/tHBaseTable/tHBaseTable_begin.javajet [hunk 2]
```diff
@@ -68,7 +69,11 @@ org.talend.bigdata.core.di.components.hbase.ImmutableTHbaseTable
                                 tableDes.addFamily(family);
                             }
                        <% } %>
-                        admin.createTable(tableDes);
+                       if (regionSplitKeysByteArr != null){
```
Comment (lbourgeois) [inline main/plugins/org.talend.designer.components.bigdata/components/tHBaseTable/tHBaseTable_begin.javajet:72]: I can't see where regionSplitKeysByteArr is declared

File: main/components_lib/talend-bigdata-core-di/src/main/java/org/talend/bigdata/core/di/components/hbase/THbaseTable.java (hunk 1 above)
Comment (sanulich) [reply main/components_lib/talend-bigdata-core-di/src/main/java/org/talend/bigdata/core/di/components/hbase/THbaseTable.java:111]: here https://github.com/Talend/tbd-studio-se/pull/3579/files#diff-0f0d23c0350d4db4ef7819c9b625466c2556a612598ab4b9b7207ee2730839f6R120

Comment (lbourgeois) [reply main/components_lib/talend-bigdata-core-di/src/main/java/org/talend/bigdata/core/di/components/hbase/THbaseTable.java:111]: Yes but that's a unit test. I would expect a call from javajet. Maybe everything is not pushed on github ?

Comment (sanulich) [reply main/components_lib/talend-bigdata-core-di/src/main/java/org/talend/bigdata/core/di/components/hbase/THbaseTable.java:111]: Yes, you are right. I have missed some changes during the merge maitenance/8.0 here. Now it's fixed

**Review by lbourgeois — APPROVED** [review]

**Pull request merged**